libqtile/backend/wayland/core.py

```diff
@@ -444,10 +445,28 @@ def _on_new_inhibitor(self, _listener: Listener, idle_inhibitor: IdleInhibitorV1
     def _on_output_power_manager_set_mode(
         self, _listener: Listener, mode: OutputPowerV1SetModeEvent
     ) -> None:
-        logger.debug("Signal: output_power_manager set_mode_event")
+        """
+        Blank/unblank outputs via the output power management protocol.
+
+        `_blanked_outputs` keeps track of those that were blanked because we don't want
+        to unblank outputs that were already disabled due to not being part of the
+        user-configured layout.
+        """
+        logger.warning("Signal: output_power_manager set_mode_event")
```

Comment:
Why change from `debug` to `warning`?

Reply:
Nice catch - I switched it to `warning` so I could debug without setting logging to debug mode locally. It should be `debug`.